src/s1reader/utils/plot_bursts.py

```diff
@@ -0,0 +1,174 @@
+import argparse
+import glob
+from importlib import import_module
+import os
+import sys
+
+named_libs = [('fiona', 'fiona'), ('folium', 'folium'), ('geopandas', 'gpd'),
+            ('pandas', 'pd')]
+for (name, short) in named_libs:
+    try:
+        lib = import_module(name)
+    except:
+        print(sys.exc_info())
+    else:
+        globals()[short] = lib
+        
```

Comment:
```suggestion

```
Trailing whitespace - 2nd issue in Codacy.

Based on your editor, trailing whitespaces can be automatically removed on save.
[VS Code](https://stackoverflow.com/questions/30884131/remove-trailing-spaces-automatically-or-with-a-shortcut)
[vim](https://stackoverflow.com/questions/356126/how-can-you-automatically-remove-trailing-whitespace-in-vim)